src/main/java/org/jenkinsci/plugins/docker/traceability/model/DockerTraceabilityReport.java

```diff
@@ -70,7 +71,7 @@ public DockerTraceabilityReport() {
     
     public DockerTraceabilityReport(@Nonnull Event event, @Nonnull Info hostInfo, 
             @CheckForNull InspectContainerResponse container, 
```

Comment:
@rsandell @jtnord 
My bad. I've thought you want me to make the containerId field nullable. Sorry for a confusion, I'll just update field annotations